.chronus/changes/improvements-for-the-intellisense-of-tspconfig.yaml-2024-11-19-2-35-2.md

```diff
@@ -0,0 +1,13 @@
+---
+# Change versionKind to one of: internal, fix, dependencies, feature, deprecation, breaking
+changeKind: fix
+packages:
+  - "@typespec/compiler"
+---
+
+Improvements for the intellisense of tspconfig.yaml
+- Supports auto-completion of the extends and imports paths
+- The rule or ruleSets of the linter can be auto-completed
+- Emitter optoins autocomplete intelligently handles quotation mark display
+- Autocomplete of variable interpolation
+- The parameters of emitter's options distinguish whether they are required or optional
```

Comment:
Please create two changelog items, one as new feature for:
Support the auto completion for extends, imports, rule, rule sets and variables in tspconfig.yaml
Show required/optional information in the details of emitter's options completion item in tspconfig.yaml
the other as bug fix for:
Fix the issue that emitter option auto completion would extra " while inside "..."